snuba/admin/static/clickhouse_queries/index.tsx

```diff
@@ -156,6 +152,40 @@ const jsonStyle = {
   borderRadius: 4,
   backgroundColor: COLORS.BG_LIGHT,
   marginBottom: 10,
+  wordBreak: "break-all" as const,
+};
+
+const executeActionsStyle = {
+  display: "flex",
+  justifyContent: "space-between",
+  marginTop: 8,
 };
 
+const executeButtonStyle = {
+  height: 30,
+  border: 0,
+  padding: "4px 20px",
+};
+
+const selectStyle = {
+  marginRight: 8,
+  height: 30,
+};
+
+function TextArea(props: {
+  value: string;
+  onChange: (nextValue: string) => void;
+}) {
+  const { value, onChange } = props;
+  return (
+    <textarea
+      spellCheck={false}
+      value={value}
+      onChange={(evt) => onChange(evt.target.value)}
+      style={{ width: "100%", height: 100 }}
+      placeholder={"Write your query here"}
```

Comment:
maybe this placeholder could be a valid query? like the one in your screenshot

Reply:
I'm just going to merge this as-is. Let's build canned queries soon 🙂